test/unit/package_manager_zypper_test.py

```diff
@@ -150,6 +150,7 @@ def test_match_package_deleted(self):
     @patch('kiwi.package_manager.zypper.RpmDataBase')
     def test_post_process_install_requests_bootstrap(self, mock_RpmDataBase):
         rpmdb = mock.Mock()
+        rpmdb.has_rpm.return_value = True
```

Comment:
Do makes sense to have another test with this to `False`, as (IIUC) in #1030?

Reply:
Thought this would fall more into the category of a real integration test. See the other commit in this PR. Being on the code path of "no rpm available" influences other behavior which come together in an integration test but are not nicely covered in the logic of a unit test. Thus I thought from a unit testing point of view the above is good enough but we have to have an integration test as well.